### PR TITLE
Feat/challenge 03 loops conditionals

### PR DESCRIPTION
**Pull request by UnPiquito**

### CODE REVIEW

File: JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    Create a program that starts with a number variable.
+    The program will return the variable is even or odd.
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+
+var numero = 7;
```
Comment (tictools) [inline JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js]: ⚠️ FIX: use 🇬🇧, please.

File: JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+} else if (age >= 13) {
+  console.log("Nice, you're a teenager!");
+} else if (age >= 18) {
+  console.log("Most places consider you an adult.");
+} else {
+  console.log("What? How did this happen!?");
+}
+
+//The problem is that the age 20, meets with all the adove conditions, thats how you can solve it:
+
```
Comment (tictools) [inline JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js]: If I execute this solution as follows...
```js
const age = 15;

if (age < 3) {
  console.log("You're just a baby!");
} else if (age < 13) {
  console.log("Nice, you're a teenager!");
} else if (age >= 18) {
  console.log("Most places consider you an adult.");
} else {
  console.log("What? How did this happen!?");
}

```
...output is `What? How did this happen!?`

Have you tested this script before pushing it? Is it properly working or is there any error in it?

⚠️ FIX: spot the 🐛 and fix it

File: JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+
+for (let i = 10; i <= 40; i++) {
```
Comment (tictools) [inline JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_01.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+/*
+    1. Create a for loop that loops while index is less or equal than 10.
+    2. Each loop:
+        * create a new random number between 1 and 10 (hint: check documentation regarding Math.floor and Math.random methods)
+        * evaluate if random number is greater than 5
+        * if it is equal or greater than 5 print "Logged In!"
+        * if it is less than 5 print "Logged Out!"
+*/
+
+for (let i = 1; i <= 10; i++) {
```
Comment (tictools) [inline JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_01.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

File: JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+for (let i = 1; i <= 100; i++) {
```
Comment (tictools) [inline JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/loops/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

This is the output of your script
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
You made it!
All, done!
```

have you tested it before pushing? 🤔

⚠️ FIX: needs work.

File: JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+let i = 1;
+
+while (i <= 100) {
```
Comment (tictools) [inline JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js]: You should take in account what `i` variable is storing. Is this variable referring to any index in this case, or a number?

⚠️ FIX: rename properly `i` variable

This is the output of your script
```
Checkpoint! 10
Checkpoint! 20
Checkpoint! 30
Checkpoint! 40
Checkpoint! 50
Checkpoint! 60
Checkpoint! 70
Checkpoint! 80
Checkpoint! 90
Checkpoint! 100
You made it!
All, done!
```

have you tested it before pushing? 🤔

⚠️ FIX: needs work.

**Review by tictools — CHANGES_REQUESTED** [review]
`node_modules` folder should never be pushed to remote repository.

In order to remove it, follow these steps:
* remove `node_modules` folder in your remote repository (branch `feat/challenge_03_loops-conditionals`)
* move to your local repo (same branch) and execute `git pull`
* fix requested changes and push them.

**Review by tictools — APPROVED** [review]

**Review by JosepRubio — APPROVED** [review]
Great job Unai!💥

**Pull request merged**